package.json

```
@@ -91,4 +92,4 @@
     "@semantic-release/npm": "10.0.6"
   },
   "packageManager": "yarn@4.0.2"
-}
+}
```

Nit: If you can configure your editor to keep/add those final `\n` :pray: 
```suggestion
}

```